Source/Engine/Physics/Actors/RigidBody.cpp

```diff
@@ -35,7 +35,11 @@ void RigidBody::SetIsKinematic(const bool value)
         return;
     _isKinematic = value;
     if (_actor)
+    {
         PhysicsBackend::SetRigidDynamicActorFlag(_actor, PhysicsBackend::RigidDynamicFlags::Kinematic, value);
+        if (!value && _isActive)
```

Comment:
Add check for `GetStartAwake()` so we don't wake up rigidbodies that are configured to favor sleeping.

Reply:
It's right now?